File: 3859-maximum-product-of-two-digits/3859-maximum-product-of-two-digits.cpp
class Solution {
public:
    int maxProduct(int n) {

        int ans = 0; 
        string s = to_string(n);
        for(int i = 0; i< s.size(); i++)
            {
                for(int j = i+1; j< s.size(); j++)
                    {
                        ans = max(ans, (s[i]-'0')*(s[j]-'0'));
                    }
            }
        return ans;
        
    }
};

File: 3859-maximum-product-of-two-digits/3859-maximum-product-of-two-digits.cs
public class Solution {
    public int MaxProduct(int n) {
         int ans = 0; 
        string s = n.ToString();
        for(int i = 0; i< s.Length; i++)
            {
                for(int j = i+1; j< s.Length; j++)
                    {
                        ans = int.Max(ans, (s[i]-'0')*(s[j]-'0'));
                    }
            }
        return ans;
        
    }
}